sw/airborne/firmwares/rotorcraft/navigation.h

```diff
@@ -59,6 +59,8 @@ extern float nav_radius;
 extern int32_t nav_leg_progress;
 extern uint32_t nav_leg_length;
 
+extern bool_t nav_survey_active;
```

Comment:
I do this because it needed to correct show the poligon on GCS.
have you another idea ?